lib/suspenders/version.rb

```diff
@@ -1,5 +1,5 @@
 module Suspenders
-  RAILS_VERSION = "~> 4.2.0"
+  RAILS_VERSION = '~> 4.2.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.